endpoints.json

```diff
@@ -89,5 +89,18 @@
         "created_at": "2024-11-26T16:26:33.344Z"
       }
     }
+  },
+  "PATCH /api/articles/:article_id": {
+    "description": "Updates the votes for specified article by a value 'newVote'. Responds with the article object with updated vote number.",
+    "exampleResponse": {
+      "article_id": 1,
+      "title": "Living in the shadow of a great man",
+      "topic": "mitch",
+      "author": "butter_bridge",
+      "body": "I find this existence challenging",
+      "created_at": "2020-07-09T20:11:00.000Z",
+      "votes": 120,
+      "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
```

Comment:
Nice, but dont forget to include the expectedBody we want to send with this patch request:

`{ "inc_votes": 1 }`